django_google_maps/static/django_google_maps/js/google-maps-admin.js

```diff
@@ -74,6 +75,19 @@ function googleMapAdmin() {
             });
         },
 
+        getMapType : function() {
+            // https://developers.google.com/maps/documentation/javascript/maptypes
+            var geolocation = document.getElementById(geolocationId);
```

Comment:
The map widget is actually tied to the address field, not geolocation. Lets change this to go after the `addressId`